recipe/meta.yaml

```diff
@@ -0,0 +1,70 @@
+{% set name = "isotree" %}
+{% set version = "0.5.24" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: d5cb06ac442f3708aa00221f917cb5d18803c9ec256b5224a1cd08935227c9ed
+  patches:
+    - patches/0001-disable-xoshiro.patch
+    - patches/0002-add-unvendored-tsl_robin_map.patch
+
+build:
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
+  number: 0
+  missing_dso_whitelist:
+    - '*/libc++.1.dylib'  # [osx]
+    - '*/libomp.dylib'    # [osx]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - llvm-openmp   # [osx]
+    - patch         # [not win]
+    - m2-patch      # [win]
+  host:
+    - python
+    - pip
+    - setuptools
+    - wheel
+    - numpy {{numpy}}
+    - cython >=3.0.0
+    - tsl_robin_map
+  run:
+    - python
+    - {{ pin_compatible('numpy') }}
+    - pandas >=0.24.0
+    - scipy
+    - _openmp_mutex  # [linux]
```

Comment:
I'm not sure if this is useful or not. @cbouss ?

Reply:
it was required, see for example https://github.com/AnacondaRecipes/scikit-learn-feedstock/blob/a7fca915fa8fadcf9ef9e38f04b87b354f95f317/recipe/meta.yaml#L47

Reply:
👀

Reply:
I'm still not sure how the openmp mutex works

Reply:
I removed and it seems to work. It was required in the previous configuration, I do not know what made this change (explicit `libgomp  # [linux]` or the CBC file)

Reply:
> I removed it and it seems to work

It's feasible that it works but when this package is installed with another package, it breaks, because of the absence of the mutex. I don't know. I think it's worth understanding what it does and _why_ you should or shouldn't remove it
If you figure out how the mutex works, it would be great to document it (maybe in the readme of the feedstock)